kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/secondChallenge/OrderService.java

```diff
@@ -10,7 +10,7 @@ public OrderService(ProductRepository productRepository) {
 
     public boolean order(Order order) {
        if (productRepository.check(order.getItem(), order.getQuantity())) {
-           productRepository.remove(order.getItem(), order.getQuantity());
+           productRepository.replace(order.getItem(), order.getQuantity());
```

Comment:
ech.. tak myslalem, ze zrobisz replace. czy repozytorium powinno miec metode replace? ze my wykorzystujemy taki mechanizm, to nie znaczy, ze mamy tak metode nazywac. tu powinna byc metoda remove wywolana.